db/Schema.dbml

```diff
@@ -135,8 +136,8 @@ table waypoints {
   sequence_id int     [note: "counts from 0 and up every time a new set of waypoints is added to a route."]
   lon float [ not null ]
   lat float [ not null ]
-  created_on timestamp [not null, default: `now()`]
-  updated_on timestamp [not null, default: `now()`, note: 'TODO: Add ON UPDATE ON UPDATE CURRENT_TIMESTAMP']
+  created_at timestamp [not null, default: `now()`]
+  updated_at timestamp [not null, default: `now()`, note: 'TODO: Add ON UPDATE ON UPDATE CURRENT_TIMESTAMP']
```

Comment:
Really nice that you are finally changing the `created_on` to `created_at`. Remember you need to update the db diagram as well. Refer to the instructions in the readme on how to.

Reply:
I have modified db diagram as well.